utam-core/src/main/java/utam/core/framework/consumer/JsonLoaderConfig.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root
+ * or https://opensource.org/licenses/MIT
+ */
+package utam.core.framework.consumer;
+
+import static utam.core.framework.consumer.UtamLoaderConfigImpl.DEFAULT_LOADER_CONFIG;
+import static utam.core.framework.consumer.UtamLoaderConfigImpl.ERR_DUPLICATE_PROFILE;
+import static utam.core.framework.context.StringValueProfile.DEFAULT_PROFILE;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import utam.core.framework.UtamCoreError;
+import utam.core.framework.context.StringValueProfile;
+
+/**
+ * JSON config for UTAM Loader public because used to write config from distribution module
+ *
+ * @author elizaveta.ivanova
+ * @since 234
+ */
+public class JsonLoaderConfig {
```

Comment:
not new file, renamed JsonBasedLoaderConfig.java

Reply:
Approved for this file.